csxtools/fastccd/images.py

```diff
@@ -44,6 +45,8 @@ def correct_images(images, dark=None, flat=None, gain=(1, 4, 8)):
     if flat is None:
         flat = np.ones(images.shape[-2:], dtype=np.float32)
         logger.info("Not correcting for flatfield. No input.")
+    else:
+        flat = np.float32(np.rot90(flat, 1))
```

Comment:
You should use http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.ndarray.astype.html